app/Console/Commands/DatabaseRefresh.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class DatabaseRefresh extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'db:refresh';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Refreshes and seeds the database for development work.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // don't run command in production or testing environment
+        if (in_array(config('app.env'), ['testing', 'production']) !== true) {
+            $this->call('migrate:fresh', ['--force' => true, '--step' => true]);
```

Comment:
What is the purpose of the `--step` flag here?

Reply:
The `--step` might be unnecessary in fresh migrations. I usually like to run migrations as stepped. Meaning each is run separately and can be rolled back one at a time rather than as a large block.